test/fuzz/lua/test_engine.lua

```diff
@@ -1024,6 +1024,24 @@ local function worker_func(id, space, test_gen, test_duration)
     return errors
 end
 
+local function disable_errinj(errinj, space)
+    local enabled_errinj = fun.iter(errinj):
+                           filter(function(i, x)
+                               if x.is_enabled then
+                                   return i
+                               end
+                           end):totable()
+    log.info(('Enabled fault injections: %s'):format(
+             json.encode(enabled_errinj)))
```

Comment:
Don't see much point in logging the list of enabled error injections at tear down but I don't really mind.

Reply:
Fixed.

```diff
     local enabled_errinj = fun.iter(errinj):
                            filter(function(i, x)
                                if x.is_enabled then
                                    return i
                                end
                            end):totable()
-    log.info(('Enabled fault injections: %s'):format(
-             json.encode(enabled_errinj)))
     for _, errinj_name in pairs(enabled_errinj) do
         local errinj_val = errinj[errinj_name].disable(space)
         errinj[errinj_name].is_enabled = false
-        log.info(string.format('DISABLE ERROR INJECTION: %s -> %s',
-                               errinj_name, tostring(errinj_val)))
         pcall(box.error.injection.set, errinj_name, errinj_val)
     end
 end
```